lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
Ah yeah makes sense. I was more trying to just get the dependencies updated and not introduce new features.